### PR TITLE
Fix overlapping banner on motions

### LINKED ISSUE #2996: Overlapping banner on motions
## Steps to reproduce

1. Create a motion
2. Ensure page length is long enough to scroll

## Expected behaviour

* Banner should be pinned to top of the content area and not overlap scroll bar.

## Actual behaviour

* Banner is overlapping the content and the scrollbar

motion-banner-overlaps

## Suggestions

* Banner should not be fixed,
* Should be fixed to the top of the screen on scroll, or
* Have the main top header stay fixed at the top of the screen along with the banner

### PR DESCRIPTION
**Pull request by danbr**
## Description

This fix locks the position of the motions banner to the top of the screen. It remains in place during scrolling.

Screenshot 2021-12-03 at 20 00 21

Resolves #2996

### CODE REVIEW

**Review by arrenv — CHANGES_REQUESTED** [review]
@danbr I have tested locally, the banner is working better now. 

The padding just needs to be corrected. See screenshots:

40554000-1387-4cb7-b689-91d90110bfe3
13d3cb38-d3de-466c-a358-6114adc02dec

98732d0d-503e-43a5-8580-b6d7bd222106
9f81ffa4-4fab-4bfa-a3c8-7a8136438238

**Review by rdig — CHANGES_REQUESTED** [review]
Holding off on my review until @arrenv 's changes are addressed

**Comment by arrenv** [comment]
Nice one! Scrolling and padding are looking much better on my end.

**Comment by arrenv** [comment]
Nice one! Scrolling and padding are looking much better on my end.

scrolling-alert-1

scrolling-alert-2

scrolling-alert-3

P.S. Sorry for closing, my internet timed out for a second as I was commenting and caused some weirdness.

**Review by arrenv — APPROVED** [review]

**Review by rdig — APPROVED** [review]
Good to go!

File: src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultAction.css [hunk 1]
```diff
@@ -50,6 +49,11 @@
 .upperContainer {
   composes: container;
   align-items: baseline;
+  padding-top: 90px;
+}
+
+.bannerPadding {
```
Comment (arrenv) [inline src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultAction.css:55]: I was thinking that bannerPadding may be better added in DefaultMotion. Although, I suppose this allows for other extensions to use it in future.

**Pull request merged**